Components/Card/card.css

```diff
@@ -0,0 +1,270 @@
+    .display-flex-row {
+        display: flex;
+        gap: 1rem;
+        flex-wrap: wrap;
+        margin-left: 5rem;
+    }
+    
+    .title-align-center {
+        margin-top: 2rem;
+    }
+    /* card with badge */
+    
+    .card-vertical {
+        max-width: 208px;
+        width: 100%;
+        position: relative;
+        box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+        margin: 1rem 1rem 0rem 1rem;
+    }
+    
+    .image-container {
+        width: 200px;
+        height: 150px;
+        padding: 1rem;
+    }
+    
+    .badge-container {
+        position: relative;
+        width: fit-content;
+        box-sizing: border-box;
+    }
+    
+    .card-container .image-container {
+        margin: 0.1rem 0;
+    }
+    
+    .card-container,
+    .card-img {
+        border-radius: 0.3rem;
+    }
```

Comment:
same as above

Reply:
yeah, will try to implement from next commits